package.json

```diff
@@ -153,6 +154,11 @@
         "command": "csharp.listProcess",
         "title": "List process for attach",
         "category": "CSharp"
+      },
+      {
+        "command" : "csharp.listRemoteProcess",
+        "title" : "List processes on remote connection for attach",
+        "category": "CSharp"
```

Comment:
`category` appears in the Command Palette:

![image](https://cloud.githubusercontent.com/assets/116161/19280352/52a3dca2-8f9a-11e6-845c-c24c1e8dd7d6.png)